ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -327,6 +327,9 @@ nnfw_tensor_type_to_gst (const NNFW_TYPE nnfw_type, tensor_type * type)
     case NNFW_TYPE_TENSOR_INT32:
       *type = _NNS_INT32;
       break;
+    case NNFW_TYPE_TENSOR_INT64:
```

Comment:
@semun-lee Please also the corresponding case in function `nnfw_tensor_type_from_gst()`.

Reply:
Thank you for your kind review. I've added it to nnfw_tensor_type_from_gst() also.